src/includes/Exports-Include.hs

```diff
@@ -49,6 +56,25 @@
   , setCursorPositionCode
 
     -- * Saving, restoring and reporting cursor position
+    -- | These code sequences are not part of ECMA-48 standard; they are popular,
+    -- but non-portable extensions. E. g., Terminal.app on MacOS
+    -- <https://stackoverflow.com/questions/25879183 does not support them>.
+    -- Somewhat surprisingly, ANSI.SYS does.
+    --
+    -- @terminfo@ package provides a portable way to determine correct sequences
+    -- for cursor positions:
+    --
+    -- >>> import System.Console.Terminfo
+    -- >>> t <- setupTermFromEnv
+    -- >>> getCapability t (tiGetOutput1 "sc") :: Maybe String -- save cursor position
+    -- Just "\ESC7"
+    -- >>> getCapability t (tiGetOutput1 "rc") :: Maybe String -- restore cursor position
+    -- Just "\ESC8"
+    --
+    -- On modern scrollable terminal emulators the semantics of cursor positions
+    -- is a bit unintuitive:
+    -- <https://unix.stackexchange.com/questions/565597 they are relative to the viewport, not to its content>.
```

Comment:
I would rather not get into what is intuitive or unintuitive, but, rather, simply explain what a user of a function needs to know to use it effectively.